pgsql/CMakeLists.txt

```diff
@@ -16,8 +16,25 @@ set ( PC_INSTALL_EXENSIONS
   "${PROJECT_BINARY_DIR}/pgsql/pointcloud.control"
   )
 
+set ( PC_SQLS
+  pointcloud.sql.in
+  )
+
+# create a "cat" function, used for concatenating the sql.in files
+# in PC_SQLS into one sql.in file
+function(cat IN_FILE OUT_FILE)
+  file(READ ${IN_FILE} CONTENTS)
```

Comment:
should `${IN_FILE}` and `${OUT_FILE}` be quoted `"${IN_FILE}"` and `"${OUT_FILE}"` ?